compiler/hsSyn/HsBinds.hs

```diff
@@ -679,7 +679,7 @@ pprDeclList :: [SDoc] -> SDoc   -- Braces with a space
 pprDeclList ds = pprDeeperList vcat ds
 
 ------------
-emptyLocalBinds :: HsLocalBindsLR (GhcPass a) (GhcPass b)
+emptyLocalBinds :: (XEmptyLocalBinds a b ~ PlaceHolder) => HsLocalBindsLR a b
```

Comment:
I'm confused about this one. It looks completely unrelated. What happened here?

Reply:
Ah, these changes snuck in when I modified the wrong data type originally. I'll remove them from this patch but they are to do with fixing https://ghc.haskell.org/trac/ghc/ticket/15084